src/main/java/org/broadinstitute/hellbender/utils/hmm/segmentation/HiddenMarkovModelPostProcessor.java

```diff
@@ -308,8 +309,9 @@ private void performSegmentation() {
      */
     private Map<String, List<HiddenStateSegment<S, T>>> calculateBestPathSegments() {
         final Map<String, List<HiddenStateSegment<S, T>>> allSegments = new LinkedHashMap<>(sampleNames.size());
-        IntStream.range(0, numSamples)
+        IntStream.range(0, numSamples).parallel()
```

Comment:
@LeeTL1220 any reason why not? parallel streams make a `ForkJoinPool` with an adaptively chosen number of threads. If the system has multiple cores, they will be all used. #GreatSpeedup